shopping-cart-client/src/app/services/login-services/login.service.ts

```diff
@@ -0,0 +1,52 @@
+import { Injectable } from '@angular/core';
+import { Subject } from 'rxjs';
+import { Register } from 'src/app/register/register.model';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class LoginService {
+  setClosingFlag = new Subject<boolean>();
```

Comment:
Does this flag belongs here?

Reply:
Modal was incomplete, it was related to that. Now it's modified